packages/block-library/src/site-logo/editor.scss

```diff
@@ -47,24 +44,27 @@
 // @todo this particular minimal style of placeholder could be componentized further.
 .wp-block-site-logo.wp-block-site-logo {
 
+	/**
 	&.is-default-size .components-placeholder {
 		height: 60px;
 		width: 60px;
 	}
+	*/
 
+	/**
 	// Inherit radius.
 	> div, // A 60px width div shown only in the editor on mobile.
 	.components-resizable-box__container {
 		border-radius: inherit;
 	}
```

Comment:
The `border-radius:inherit` on the `>div` here is overriding the border radius from the global styles.
I did not understand the code comment about the div only shown in the editor on mobile or how to test it.